main.js

```diff
@@ -78,6 +88,29 @@ const speakers = [
 ];
 
 /* -----------------DYNAMIC SECTION CREATION --------------*/
+const speakersBtn = document.querySelector('.featured__section__btn');
+
+function moreSpeakers() {
+  const speakers = document.querySelectorAll('.featured__section__wrapper > :nth-child(n+3)');
+
+  const speakersStatus = window.getComputedStyle(speakers[0]).display;
+
+  if (speakersStatus === 'none') {
+    speakers.forEach((speaker) => {
+      speaker.style.display = 'grid';
+    });
+    speakersBtn.textContent = 'LESS';
+    speakersBtn.innerHTML += '<i class="fa-solid fa-chevron-up"></i>';
+  } else {
+    speakers.forEach((speaker) => {
+      speaker.style.display = 'none';
+    });
+    speakersBtn.textContent = 'MORE';
+    speakersBtn.innerHTML += '<i class="fa-solid fa-chevron-down"></i>';
+  }
+}
```

Comment:
- [ ] [OPTIONAL] Great job on implementing the `more` button to mobile menu for adding more speakers however please make sure that the desktop version has all the speakers displayed since it doesn't have `more` button. Also to stick to the design and improve user experience as the desktop version is supposed to be the most convenient and wide. 

|Cindy's design|Your design|
|----|----|
|<img width="742" alt="Screenshot 2023-05-25 at 10 49 47 AM" src="https://github.com/sergio-usma/First_capstone_project/assets/44798044/ae18893e-dea7-4378-98bf-80f8f7b3715b">|<img width="1161" alt="Screenshot 2023-05-25 at 10 48 21 AM" src="https://github.com/sergio-usma/First_capstone_project/assets/44798044/6850cfa2-e5b2-4cf3-bead-e2c90d7d82fd">|